### PR TITLE
Fix #708 by restoring `<head>` to `baseof.html` with template structure

### LINKED ISSUE #708: 3762d3a is breaking title generation
Hi, reporting an issue for now and trying to dig deeper to fix it.

Since commit 3762d3a the title is not generated correctly anymore. Instead of rendering the page title followed by the blog title, only the blog title is getting rendered.

/cc @fj

### PR DESCRIPTION
**Pull request by fj**
### Prerequisites

Put an `x` into the box(es) that apply:

- [x] This pull request fixes a bug.
- [ ] This pull request adds a feature.
- [ ] This pull request introduces breaking change.

### Description

Fixes #708.

### Issues Resolved

#708 

### Checklist

Put an `x` into the box(es) that apply:

#### General

- [x] Describe what changes are being made
- [x] Explain why and how the changes were necessary and implemented respectively
- [x] Reference issue with `#<ISSUE_NO>` if applicable

Fixes #708 by moving `<head>` back into `baseof.html` while preserving the template structure.

#### Resources

- [x] If you have changed any SCSS code, run `make release` to regenerate all CSS files

#### Contributors

- [x] Add yourself to `CONTRIBUTORS.md` if you aren't on it already

### CODE REVIEW

**Comment by luizdepra** [comment]
I'm going to make some change here. Nothing to worry about.

File: layouts/_default/baseof.html [hunk 1]
```diff
@@ -1,6 +1,9 @@
 <!DOCTYPE html>
 <html lang="{{ .Site.Language.Lang }}">
-{{ partial "head.html" . }}
+  <head>
+    <title>{{ block "title" . }}site: {{ .Site.Title }}{{ end }}</title>
```
Comment (luizdepra) [inline layouts/_default/baseof.html:4]:
```suggestion
    <title>{{ block "title" . }}{{ .Site.Title }}{{ end }}</title>
```

**Comment by luizdepra** [comment]
Thank you. :)

**Comment by fj** [comment]
Thanks for catching my error @luizdepra. @truemped can you confirm all is well now?

**Comment by truemped** [comment]
Yes, thanks a mil! Works again and like a charm!

**Comment by truemped** [comment]
This blog post triggered everything: https://truemped.github.io/posts/mgmt/buy-vs-build/

And I see my title again!

:D

**Pull request merged**